### ISSUE TITLE
merging when travis failed

### ISSUE THREAD

**Issue by matthewdking** [veredrec/The-Magic-VASS#53]
even when travis failed to build the branch was still merged with master

![screenshot from 2017-08-03 10-28-32](https://user-images.githubusercontent.com/25408167/28910717-a8cea9d8-7836-11e7-9fb3-c2bf85febb6e.png)

**Comment by veredrec** [comment]
Good point @matthewdking! We will make sure to check it from now on before merging :)
